playbooks/test_deploy_cukinia_tests.yaml

```diff
@@ -36,5 +40,41 @@
         src: ../src/debian/vm_manager/vm_manager/testdata/wrong_vm_config.xml
         dest: /usr/share/testdata
 
-
+- name: Deploy Cukinia's tests
+  hosts: observers
+  tasks:
+    - name: Copy Cukinia's tests
+      copy:
+        src: ../src/cukinia-tests/cukinia
+        dest: /etc/
+    - name: Copy cukinia.conf
+      template:
+        src: ../templates/cukinia-observer.conf.j2
+        dest: /etc/cukinia/cukinia.conf
+    - name: Create /usr/share/cukinia/includes
```

Comment:
Cukinia includes should be copy in hypervisors too.

Reply:
They already are